Post/migrations/0002_auto_20210406_1033.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.7 on 2021-04-06 10:33
```

Comment:
Why this is 0002_auto generated? 
If you run migration on clean db, it will not regenerated

Reply:
At first, I configured the 'photoURL' field to be a CharField with max length of 100 chars.
It wasn't the right way to configure that field, so I changed it to TextField and ran the 'makemigrations' command so it will be updated.

Reply:
I would advise to have the correct format here 
Post/migrations/0001_initial.py

Please, clean the db.sqlite3 and rerun make migration, this is how it will work on ci testing.
Upload the updated 0001_initial.py
and git rm the auto_file

This is also the reason why it is recommended to add test.py in migrations directory similar to 1st challenge

Reply:
One more comment here,
Please suggest test_data under migration dir.
similar to Task#5 in 1st challenge.

Reply:
I made the changes to the migration files as you asked.

Reply:
I added a test_data file.